contracts/interfaces/external/IERC20PermitAllowed.sol

```diff
@@ -1,10 +1,19 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity >=0.5.0;
 
+/// @title Interface for permit
 /// @notice Interface used by DAI/CHAI for permit
 interface IERC20PermitAllowed {
     /// @notice Approve the spender to spend some tokens via the holder signature
     /// @dev This is the permit interface used by DAI and CHAI
+    /// @param holder the address of the token holder, the token owner
```

Comment:
be sure to capitalize the beginning of a comment, i.e., `@param holder The address of the token holder`